src/isa/riscv64/Kconfig

```diff
@@ -86,6 +90,7 @@ config MARCHID_VALUE
   depends on RV_ARCH_CSRS
   depends on !USE_XS_ARCH_CSRS
   hex "Value of CSR marchid"
+  default 0x5 if DIFFTEST_REF_SPIKE
```

Comment:
Does the csr has multiple default value?

Reply:
When we open difftest with Spike（as ref）, some workload, for example, opensbi + linux, will read marchid to 'a0',  so we need to change the value of marchid of NEMU when difftest with Spike. However, I will change marchid in isa/riscv64/init.c otherwise Kconfig.